src/utils/viewer/viewer.cpp

```diff
@@ -391,8 +418,34 @@ void Viewer::drawEvent() {
   if (sceneID_.size() <= 0)
     return;
 
-  if (physicsManager_ != nullptr)
+  imgui_.newFrame();
+
+  if (physicsManager_ != nullptr) {
+    if (objectIDs_.size() > 0) {
```

Comment:
Then is a `showFPS` to enable and disable the info displayed on the screen.
Would you consider using it here?